lib/c_glib/src/thrift/c_glib/transport/thrift_ssl_socket.c

```diff
@@ -270,10 +274,11 @@ thrift_ssl_socket_open (ThriftTransport *transport, GError **error)
 gboolean
 thrift_ssl_socket_close (ThriftTransport *transport, GError **error)
 {
-  gboolean retval = FALSE;
+  /* gboolean retval = FALSE; */
   ThriftSSLSocket *ssl_socket = THRIFT_SSL_SOCKET(transport);
   if(ssl_socket!=NULL && ssl_socket->ssl) {
       int rc = SSL_shutdown(ssl_socket->ssl);
+      (void)rc;
```

Comment:
Better to remove it entirely if it's unused.

I did this in #1830 but still waiting on approval. cc @Jens-G

Reply:
Done.